tasks/rubies.yml

```diff
@@ -62,16 +62,24 @@
   when: not '--user-install' in rvm1_install_flags and rvm1_bundler_install
   with_items: '{{ rvm1_symlink_bundler_binaries }}'
 
-- name: List installed versions of Ruby (for deletion)
+- name: List installed ruby versions (for deletion)
   command: '{{ rvm1_rvm }} list strings'
   changed_when: False
   register: rvm1_list_installed_rubies
-  when: rvm1_delete_ruby is defined and rvm1_delete_ruby != ''
+  when: rvm1_delete_unmanaged_rubies|bool or (rvm1_delete_ruby is defined and rvm1_delete_ruby != '')
```

Comment:
maybe we should avoid the sometimes confusing `bool` filter and just use `true/false` or `yes/no`?

Reply:
Sure, I totally agree, at this level we must assume that the variable type is of the correct type.

I initially wanted to ensure that the variable was converted as a boolean, for example to allow bare variables to be passed as simple `--extra-vars` arguments. But I don't like the idea to be forced to put defensive filter everywhere just for this "comfort".

Reply:
Fixed in fada459a67fc67e0d5a11dee507d2ab76ecc7a3a